android/tooling/rib-intellij-plugin/src/main/resources/templates/kotlin/RibBuilder.kt.template

```diff
@@ -0,0 +1,88 @@
+package ${package_name};
+
+import com.uber.rib.core.Builder
+import com.uber.rib.core.EmptyPresenter
+import com.uber.rib.core.InteractorBaseComponent
+import java.lang.annotation.Retention
+
+import javax.inject.Qualifier
+import javax.inject.Scope
+
+import dagger.Provides
+import dagger.BindsInstance
+
+import java.lang.annotation.RetentionPolicy.CLASS
+
+class ${rib_name}Builder(dependency: ParentComponent) : Builder<${rib_name}Router, ${rib_name}Builder.ParentComponent>(dependency) {
+
+    /**
+     * Builds a new [${rib_name}Router].
+     *
+     * @return a new [${rib_name}Router].
+     */
+    fun build(): ${rib_name}Router {
+        val interactor = ${rib_name}Interactor()
+        val component = Dagger${rib_name}Builder_Component.builder()
+                .parentComponent(dependency)
+                .interactor(interactor)
+                .build()
+
+        return component.${rib_name_to_lower}Router()
+    }
+
+    interface ParentComponent// TODO: Define dependencies required from your parent interactor here.
```

Comment:
Can you put your TODOs on the line above in order to match Uber's open source code style?

Reply:
Sure thing. Thanks